lib/tochtli/reply_queue.rb

```diff
@@ -56,7 +56,7 @@ def register_message_handler(message, handler=nil, timeout=nil, &block)
 
     def on_delivery(delivery_info, metadata, payload)
       class_name       = metadata.type.camelize.gsub(/[^a-zA-Z0-9\:]/, '_') # basic sanity
-      reply_class      = eval(class_name)
+      reply_class      = Object.const_get(class_name)
```

Comment:
This is not the same, as far as I know. With `Object.const_get` we can only obtain classes from global namespace, but we might want (actually, we DO want) to have access top namespaced classes as well.